### PR TITLE
feat : mobile responsive

### PR DESCRIPTION
**Pull request by mikesch803**
1. add mobile responsive
2. bug fix in cart page

### CODE REVIEW

File: src/components/aside/Aside.jsx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import {
   filterRatingData,
 } from "../../data/filter-data/filter.data";
 import "./Aside.css";
-export function Aside() {
+export function Aside({ asideStyles }) {
   const { dispatch, state } = useFilter();
 
   return (
-    <aside className="aside-product">
+    <aside className="aside-product" style={{ display: asideStyles }}>
```
Comment (vrx29) [inline src/components/aside/Aside.jsx:13]: avoid using inline style

File: src/components/cart-price-card/CartPriceCard.jsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export function CartPriceCard() {
       {state.cartData.length !== 0 && (
         <div className="cart-price-detail">
           <h3 className="txt-left">Price details</h3>
-          <div className="card card-vrt card-txt">
+          <div className="card card-vrt card-txt card-price">
             <p>
               Price ({state.cartData.length}){" "}
```
Comment (vrx29) [inline src/components/cart-price-card/CartPriceCard.jsx:18]: remove this extra data {""}

**Review by vrx29 — APPROVED** [review]

File: src/components/aside/Aside.jsx (hunk 1 above)
Comment (mikesch803) [reply src/components/aside/Aside.jsx:13]: i used this to show filter in mobile view

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Code Looks Clean and Readable Nice Work

**Pull request merged**